workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -13,7 +14,8 @@ protected override ReceptionistConfig GetReceptionistConfig(string workerType)
                 ReceptionistHost = GetHostIp(),
                 WorkerType = workerType,
                 WorkerId = CreateNewWorkerId(workerType),
-                UseExternalIp = UseExternalIp
+                UseExternalIp = UseExternalIp,
+                LinkProtocol = NetworkConnectionType.Tcp,
```

Comment:
would this be configurable too, or is it only working in Tcp anyway?

Reply:
they can overwrite it. but raknet is very unstable and not really supposed to work with mobile

Reply:
👍